Makefile

```
@@ -268,6 +279,24 @@ KUTTL = $(shell which $(KUTTL))
 endif
 endif
 
+# Download crane binary if necessary
+CRANE_RELEASE = v0.14.0
+CRANE = $(shell pwd)/bin/crane
+CRANE_DL_URL = https://github.com/google/go-containerregistry/releases/download/$(CRANE_RELEASE)/go-containerregistry_$(OS_CAP)_$(ARCH).tar.gz
```

I assume you're running this on an arm box, this doesn't work on my x86 box: `ARCH` value is `amd64`, but crane binary is named `x86_64` (see [latest release binaries](https://github.com/google/go-containerregistry/releases/tag/v0.14.0)), hence this change is needed to make it work:
```suggestion
CRANE_DL_URL = https://github.com/google/go-containerregistry/releases/download/$(CRANE_RELEASE)/go-containerregistry_$(OS_CAP)_$(shell uname -m).tar.gz
```